btree/btree.c

```diff
@@ -1071,11 +1071,25 @@ struct node_type {
 	int  (*nt_create_delete_credit_size)(void);
 
 	/**
-	 * Calculates credits required to free the node and adds those credits
-	 * to @accum
+	 * Calculates credit required for the put KV operation and adds those
+	 * credit to @cred.
 	 */
-	void (*nt_node_free_credits)(const struct nd *node,
-				     struct m0_be_tx_credit *accum);
+	void (*nt_put_credit) (const struct nd *node, m0_bcount_t ksize,
+			       m0_bcount_t vsize, struct m0_be_tx_credit *cred);
+
+	/**
+	 * Calculates credit required for the update KV operation and adds those
+	 * credit to @cred.
+	 */
+	void (*nt_update_credit) (const struct nd *node, m0_bcount_t ksize,
+			       m0_bcount_t vsize, struct m0_be_tx_credit *cred);
+
+	/**
+	 * Calculates credit required for the update KV operation and adds those
```

Comment:
/s/update/del

Reply:
Made changes in code after this comment ,  this part is outdated now.